### ISSUE TITLE
PhotoBrowser - GoogleClientLogin - Cancel button

### ISSUE THREAD

**Issue by GoogleCodeExporter** [barsu8184/google-gdata#122]
```
What steps will reproduce the problem?
1. Open GoogleClientLogin
2. Click in Cancel button
3. Thus the error occurs

What is the expected output? What do you see instead?
The output was expected to be closed application without any problem.
The problem is that instead leave the program, he tries to do the same 
authentication not having the data. With this peace of 
code:picasaService.SetAuthenticationToken(loginDialog.AuthenticationToken);

My solution:
if(loginDialog.DialogResult == DialogResult.OK) {
   this.googleAuthToken = loginDialog.AuthenticationToken;
   this.user = loginDialog.User;

   if (this.googleAuthToken == null) {
       this.Close();
   }
   else {
         picasaService.SetAuthenticationToken           
(loginDialog.AuthenticationToken);
         UpdateAlbumFeed();
    }
}
else {
   Application.Exit();
}


And a verification on Login Button:
if (Username.Text == "")
{
  MessageBox.Show("Please specify a username", "No user name", 
MessageBoxButtons.OK, MessageBoxIcon.Error);
                return;
}
if (Password.Text == ""){
   MessageBox.Show("Please specify a password", "No password", 
MessageBoxButtons.OK, MessageBoxIcon.Error);
   return;
}

What version of the product are you using? On what operating system?
Google Data API SDK(1.1.4.1). Windows Vista Home Premiun.


Please provide any additional information below.

```

Original issue reported on code.google.com by `thiago.f...@gmail.com` on 7 May 2008 at 2:56

**Comment by GoogleCodeExporter** [comment]
```
I did already change the login dialog used in the project templates to have a 
correct behaviour. I need to fix the 
browser, and probably move the login into the core dll to be shared....
```

Original comment by `fman...@gmail.com` on 8 May 2008 at 10:07
- Changed state: **Reviewed**

**Comment by GoogleCodeExporter** [comment]
```
You can create a project for Login System, then share with all other sample 
applications.

Because most of samples need to login in google account, and if you don't put 
the 
information correct the application crash.
```

Original comment by `thiago.f...@gmail.com` on 9 May 2008 at 10:47

**Comment by GoogleCodeExporter** [comment]
```
I fixed the photobrowser code. I do assume most people use the project 
templates, or are at least able to copy 
this out of there. I will probably move this to another dll for reference on 
the longer run.

The fix will be in the 1.2.0.0 release and up
```

Original comment by `fman...@gmail.com` on 5 Jun 2008 at 8:18
- Changed state: **Fixed**
